source/common/http/utility.h

```diff
@@ -153,6 +153,13 @@ class Utility {
    *         @see GetLastAddressFromXffInfo for more information.
    */
   static GetLastAddressFromXffInfo getLastAddressFromXFF(const Http::HeaderMap& request_headers);
+
+  /**
+   * Get the string for the given http protocol.
+   * @param protocol for which to return the string representation.
+   * @return string representation of the protocol.
+   */
+  static const std::string& getProtocolString(const Protocol p);
```

Comment:
By docs comments I think this should be named "protocol" rather than "p"

Reply:
oops. fixed this and the other comment as well.
Merged master also, hopefully that will help with the macos build. fingers crossed.
Thanks for the quick review.